components/climate/thermostat.rst

```diff
@@ -153,6 +153,9 @@ These temperatures are used when the device first starts up.
 **At least one of** ``default_target_temperature_low`` **and** ``default_target_temperature_high``
 **must be specified.**
 
+``min_temperature`` and ``max_temperature`` from the base climate component **will limit the range**
+of allowed temperature values in the thermostat component.  See :doc:`/components/climate/index`
+
```

Comment:
I guess the only additional comment I would have is to perhaps add a sentence to address the people who ask "why"? For example:
_"These values are used to ensure any given set point is **always** within the desired range -- you wouldn't  want to set your living room's thermostat to -999999 degrees in the middle of winter, would you?"_

Totally optional and absolutely not necessary...but...we're engineers...and inevitably people jump on the discord server and ask "why?"

Reply:
thanks see update